.github/workflows/release.yaml

```diff
@@ -138,3 +138,95 @@ jobs:
           git add .SRCINFO PKGBUILD
           git commit -m "Update to $(grep '^pkgver=' PKGBUILD | cut -d'=' -f2)"
           git push origin master
+
+  publish-to-debian-bookworm:
+    name: 🐧 Generate debian package
+    needs: release
+    if: needs.release.outputs.should_publish == 'yes'
+    runs-on: ubuntu-latest
+
+    container: 
+      image: debian:bookworm
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Install basic dependencies
+        id: install-software
+        run: |
+          export DEBIAN_FRONTEND=noninteractive
+          apt-get update
+          apt-get install -y wget git lsb-release wget software-properties-common gnupg curl build-essential devscripts debhelper
```

Comment:
I would prefer if we put these steps inside a `Dockerfile` and adding a new target in our `Makefile` to build the debian package so it's easy for us to build the deb package without the GitHub action.

This also makes it easier for us to switch to a different CI solution other than GitHub Actions if we ever need to do so.

Reply:
I have created a dist-deb very similar to the dist-rpm.
I was unable to make use of an in-repo Dockerfile without giving direct docker commands. I personally find that a bit ugly and unnecessary given dependencies are listed there, but if you want that I can put those command in the github CI, just be aware the file won't look particularly nice afterward

Reply:
What is the plan for this?